### PR TITLE
v2.11.1 Changes Staging merge

### PR DESCRIPTION
**Pull request by selinaarokiaswamy-GSLAB**

### CODE REVIEW

**Comment by selinaarokiaswamy-GSLAB** [comment]
@JosephPJordan Could you please review and merge this pull request to staging? Thanks

File: docs/l4-l7.rst [hunk 1]
```diff
@@ -61,7 +61,7 @@ Create a partition using Advanced mode (Application subtab)
 
 5. Update the stub code with valid inputs.
 
-6. Optionally, Click on :guilabel:`Manage Endpoint Mappings` to add Dynamic Endpoint Mappings for endpoint discovery. You can select APIC Tenant, Application, Endpoint Group from the provided drop-downs to associate them with the Application’s pool. The VRF field is a read-only field and is retrieved from the APIC’s Endpoint Group itself. You can select Route Domain from the dropdown options to associate them with the Application's pool. Whenever the APIC Endpoint Group will have an endpoint list updated, the AS3 application’s pool will also have the same set of endpoints in it on the BIG-IP. 
+6. Optionally, click on :guilabel:`Manage Endpoint Mappings` to configure Dynamic Endpoint Mappings for endpoint discovery, which dynamically updates endpoints in BIG-IP pools whenever endpoints get attached or detached from APIC Endpoint Groups.
```
Comment (JosephPJordan) [inline docs/l4-l7.rst:64]: Optionally, click :guilabel:`Manage Endpoint....
...whenever endpoints are attached or detached...

File: docs/l4-l7.rst [hunk 2]
```diff
@@ -130,20 +132,21 @@ Update an application using Advanced mode (Application subtab)
    to update.
 
    .. note::
-      Application contents is retrieved through AS3 APIs from the BIG-IP device and displayed in the text box.
+      Application content is retrieved through AS3 APIs from the BIG-IP device and displayed in the text box.
 
 4. Update the application JSON.
 
-5. Optionally, click :guilabel:`Manage Endpoint Mappings` to update Dynamic Endpoint Mappings for endpoint discovery. You can select APIC Tenant, Application, Endpoint Group from the provided drop-downs to associate them with the Application’s pool.The VRF field is a read-only field and is retrieved from the APIC’s Endpoint Group itself. You can select Route Domain from the dropdown options to associate them with the Application's pool. Whenever the APIC Endpoint Group will have an endpoint list updated, the AS3 application’s pool will also have the same set of endpoints in it on the BIG-IP. 
+5. Optionally, click :guilabel:`Manage Endpoint Mappings` to configure Dynamic Endpoint Mappings for endpoint discovery, which dynamically updates endpoints in BIG-IP pools whenever endpoints get attached or detached from APIC Endpoint Groups.
```
Comment (JosephPJordan) [inline docs/l4-l7.rst:139]: ... are attached or detached

File: docs/l4-l7.rst [hunk 3]
```diff
@@ -130,20 +132,21 @@ Update an application using Advanced mode (Application subtab)
...
 6. Click :guilabel:`Submit`.
 
 The application is updated on the BIG-IP device.
 
 .. note::
 
-   - Route Domains must be created on the BIG-IP since the L2-L3 Network Management Tab, as well as AS3 configuration, does not support creation of Route Domains.
    - Minimum supported AS3 version for the VRF to Route Domain mapping support is v3.41.
+   - Route Domains must be created on the BIG-IP.  FASC does not support the creation of Route Domains.
+   - :guilabel:`Manage Endpoint Mappings` APIC VRF field is read-only and is retrieved from the APIC based on the provided info.
```
Comment (JosephPJordan) [inline docs/l4-l7.rst:149]: ... information.

File: docs/l4-l7.rst [hunk 4]
```diff
@@ -190,20 +193,22 @@ New partitions and applications are listed in the Partition and Application drop
 
 .. note::
 
-   - On clicking :guilabel:`Submit`, the partition is created asynchronously in the background. Once the partition is created successfully, the app UI loads the latest data. For more details, refer to https://clouddocs.f5.com/f5-aci-servicecenter/latest/l4-l7.html#as3-async-task-processing
-   
-   - For the :guilabel:`service-discovery/http` template, you can select APIC Tenant, Application, Endpoint Group from the provided drop-downs to associate them with the BIG-IP Application’s pool. The VRF field is a read-only field and is retrieved from the APIC’s Endpoint Group itself. You can select Route Domain from the drop-down lists to associate them with the Application's pool. Whenever the APIC Endpoint Group will have a new endpoint attached or detached from it, the FAST application’s pool will also have the same set of endpoints in it on the BIG-IP.
+   - Minimum supported AS3 version for VRF to Route Domain mapping is v3.41.
+   -  :guilabel:`Service-discovery/http template` provides endpoint discovery, which dynamically updates endpoints in BIG-IP pools whenever endpoints get attached or detached from APIC Endpoint Groups.
```
Comment (JosephPJordan) [inline docs/l4-l7.rst:197]: ..are attached..

File: docs/l4-l7.rst [hunk 5]
```diff
@@ -239,10 +244,12 @@ The new application is created on this BIG-IP device in the specified partition,
...
-   - For the :guilabel:`service-discovery/http` template, you can select APIC Tenant, Application, Endpoint Group from the drop-downs lists to associate them with the BIG-IP Application’s pool. The VRF field is a read-only field and is retrieved from the APIC’s Endpoint Group itself. You can select Route Domain from the dropdown options to associate them with the Application's pool. Whenever the APIC Endpoint Group will have a new endpoint attached or detached from it, the FAST application’s pool will also have the same set of endpoints in it on the BIG-IP.
-   - Route Domain must be created on the BIG-IP since the L2-L3 Network Management Tab as well as AS3 configuration does not support creation of Route Domains.
-   - Minimum supported AS3 version for the VRF to Route Domain mapping support is v3.41.
+   - Minimum supported AS3 version for the VRF to Route Domain mapping support is v3.41. 
+   - :guilabel:`Service-discovery/http` template provides endpoint discovery, which dynamically updates endpoints in BIG-IP pools whenever endpoints get attached or detached from APIC Endpoint Groups.
```
Comment (JosephPJordan) [inline docs/l4-l7.rst:248]: ...are attached...

File: docs/l4-l7.rst [hunk 6]
```diff
@@ -346,15 +353,15 @@ View AS3/FAST Applications (Application Inventory sub-tab)
...
+   - Displays the application name. Each application name in this table has a hyperlink to the Application sub-tab to the respective mode (either Basic/Advanced) used during creation of the application.
 
    **Template**    
    
    - Displays the name of the template used. If the application was created using Advanced mode, this column will display '-'.
 
    **Pool**    
    
-   - BIG-IP pool names which are present in this application. Each pool should be displayed on a separate row. The pool has hyperlink to BIG-IP UI to the same pool.
+   - BIG-IP pool names which are present in this application. Each pool should be displayed on a separate row. The pool has a hyperlink to BIG-IP UI to the same pool.
```
Comment (JosephPJordan) [inline docs/l4-l7.rst:364]: ... to the BIG-IP UI...

File: docs/l4-l7.rst [hunk 7]
```diff
@@ -388,10 +395,10 @@ View AS3/FAST Applications (Application Inventory sub-tab)
    - View Application JSON icon - Shows the raw JSON of the AS3 application
 
 
-Refresh L4-L7 Configuration tab
+Refresh L4-L7 App Services tab
```
Comment (JosephPJordan) [inline docs/l4-l7.rst:398]: Refresh the...

File: docs/release-notes.rst [hunk 8]
```diff
@@ -4,13 +4,36 @@ What's New in Version 2.11
...
+
+**1. VIP Visibility Dashboard does not display APIC information for dynamically learned VIPs**
+
+VIP Visibility Dashboard only displays APIC information for VIPs that are defined as static endpoints on the APIC.  VIP Visibility Dashboard does not display APIC information for VIPs that are dynamically learned on the APIC.
+
+------
+
+**2. VIP and VLAN Visibility Table does not display APIC VRF information for dynamically learned VIPs**
+
+VIP and VLAN Visibility Table does not display VRF information if the VIP is dynamically learned on the APIC.  VRF information is displayed if the VIP is defined as a static endpoint on the APIC.
```
Comment (JosephPJordan) [inline docs/release-notes.rst:26]: The VIP and VLAN...

**Review by JosephPJordan — CHANGES_REQUESTED** [review]
A few minor suggested changes

**Review by JosephPJordan — APPROVED** [review]

**Pull request merged**